src/Elastic.OpenTelemetry/Configuration/ElasticOpenTelemetryOptions.cs

```diff
@@ -24,79 +27,121 @@ namespace Elastic.OpenTelemetry.Configuration;
-				ref _enabledElasticDefaultsSource, StringParser);
-
-			BindFromLoggingSection(configuration);
-		}
+		if (configuration is null)
+			return;
+		SetFromConfiguration(configuration, LogDirectoryConfigPropertyName, ref _logDirectory, ref _logDirectorySource, StringParser);
+		SetFromConfiguration(configuration, LogLevelConfigPropertyName, ref _logLevel, ref _logLevelSource, LogLevelParser);
+		SetFromConfiguration(configuration, LogTargetsConfigPropertyName, ref _logTargets, ref _logTargetsSource, LogTargetsParser);
+		SetFromConfiguration(configuration, SkipOtlpExporterConfigPropertyName, ref _skipOtlpExporter, ref _skipOtlpExporterSource, BoolParser);
+		SetFromConfiguration(configuration, EnabledElasticDefaultsConfigPropertyName, ref _enabledElasticDefaults, ref _enabledElasticDefaultsSource, StringParser);
+
+		BindFromLoggingSection(configuration);
 
-		void BindFromLoggingSection(IConfiguration configuration)
+		void BindFromLoggingSection(IConfiguration config)
 		{
 			// This will be used as a fallback if a more specific configuration is not provided.
 			// We also store the logging level to use it within the logging event listener to determine the most verbose level to subscribe to.
-			_loggingSectionLogLevel = configuration.GetValue<string>($"Logging:LogLevel:{CompositeLogger.LogCategory}");
+			_loggingSectionLogLevel = config.GetValue<string>($"Logging:LogLevel:{CompositeLogger.LogCategory}");
 
 			// Fall	back to the default logging level if the specific category is not configured.
 			if (string.IsNullOrEmpty(_loggingSectionLogLevel))
-				_loggingSectionLogLevel = configuration.GetValue<string>("Logging:LogLevel:Default");
+				_loggingSectionLogLevel = config.GetValue<string>("Logging:LogLevel:Default");
 
-			if (!string.IsNullOrEmpty(_loggingSectionLogLevel) && _fileLogLevel is null)
+			if (!string.IsNullOrEmpty(_loggingSectionLogLevel) && _logLevel is null)
 			{
-				_fileLogLevel = _loggingSectionLogLevel;
-				_fileLogLevelSource = ConfigSource.IConfiguration;
+				_logLevel = LogLevelHelpers.ToLogLevel(_loggingSectionLogLevel);
+				_logLevelSource = ConfigSource.IConfiguration;
 			}
 		}
 	}
 
+	/// <summary>
+	/// Calculates whether global logging is enabled based on
+	/// <see cref="LogTargets"/>, <see cref="LogDirectory"/> and <see cref="LogLevel"/>
+	/// </summary>
+	public bool GlobalLogEnabled
+	{
+		get
+		{
+			var isActive = (_logLevel.HasValue || !string.IsNullOrWhiteSpace(_logDirectory) || _logTargets.HasValue);
+			if (isActive)
+			{
+				if (_logLevel is LogLevel.None)
+					isActive = false;
+				else if (_logTargets is LogTargets.None)
+					isActive = false;
+			}
+			return isActive;
+		}
+	}
+
+	private static string GetDefaultLogDirectory() =>
+		Environment.OSVersion.Platform == PlatformID.Win32NT
+		? Path.Combine(Environment.GetEnvironmentVariable("PROGRAMDATA")!, "elastic", "apm-agent-dotnet", "logs")
```

Comment:
Any reason not to use something like `Environment.SpecialFolder.LocalApplicationData` here?